recipes/open62541/all/test_package/conanfile.py

```diff
@@ -6,12 +6,20 @@ class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     generators = "cmake", "cmake_find_package_multi"
 
+    def requirements(self):
+        self.requires("ua-nodeset/1.02-2021-07-21")
+
     def build(self):
         cmake = CMake(self)
+        cmake.definitions["open62541_NODESET_DIR"] = self.deps_user_info["ua-nodeset"].nodeset_path
```

Comment:
```suggestion
        cmake.definitions["open62541_NODESET_DIR"] = self.deps_user_info["ua-nodeset"].nodeset_dir
```

See https://github.com/conan-io/conan-center-index/pull/7312/files#r712805774